src/main/java/com/sap/cloudfoundry/client/facade/CloudControllerClient.java

```diff
@@ -645,18 +529,6 @@ void createApplication(String applicationName, Staging staging, Integer disk, In
      */
     List<CloudStack> getStacks();
 
-    /**
-     * Login using the credentials already set for the client.
-     *
-     * @return authentication token
-     */
-    OAuth2AccessTokenWithAdditionalInfo login();
```

Comment:
What is used in DataTerminationService and OrphanedDataCleaner when there is no login

Reply:
In `DataTerminationService` we use a custom controller client, which creates its `WebClient` with an initialized `OAuthClient` from the credentials passed to it.
In `OrphanedDataCleaner` we do a similar thing, where we create a `CloudSpaceClient` with a new, initialized `OAuthClient`